### DISCUSSION TITLE
Doc comment on REST - /v1/modules (/api/rest/modules)

### DISCUSSION THREAD

**Discussion by giscus** [weaviate/weaviate-io#1767] [Docs/blog comments]
# Doc comment on REST - /v1/modules (/api/rest/modules)

The /v1/modules endpoint is available for all modules that are connected to your Weaviate instance. The available module endpoints thus depend on how you configured your weaviate. For API references of specific modules offered by Weaviate, visit the specific modules pages, for example the text2vec-contextionary.

https://weaviate.io/developers/weaviate/api/rest/modules

**Comment by Gigadelux** [comment]
guarda qua

**Comment by lucaspeyrin** [comment]
So can you set an API key through this endpoint on a WCS ?

For example the OPENAI_APIKEY ?
